<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiTransport</class>
 <widget class="QWidget" name="UiTransport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>67</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IanniX â€” Transport</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	font: 12px &quot;Arial&quot;;
	color: white;
}

QFrame#globalFrame {
	border: 0px solid transparent;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(74, 74, 74, 255), stop:1 rgba(22, 22, 22, 255));
}
QFrame#transportFrame, QFrame#timeFrame, QFrame#optionFrame, QFrame#speedFrame, QFrame#logoFrame, QFrame#perfFrame {
	border: 0px solid transparent;
	border-left: 1px solid rgb(80,80,80);
}


QPushButton, QDoubleSpinBox {
	border: 1px solid rgb(60, 60, 60);
	border-radius: 4px;
	color: rgb(192, 192, 192);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(99, 100, 103, 255), stop:1 rgba(61, 61, 61, 255));

}
QPushButton:hover {
	border: 1px solid rgb(0, 187, 255);
	background-color: rgb(0, 137, 241, 128);
}
QPushButton::checked, QPushButton:pressed {
	background-color: rgb(0, 187, 255);
	color: white;
}


QLineEdit#timeEdit {
	font: 12px &quot;Lucida Sans&quot;, &quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, &quot;Lucida&quot;;
	font-size: 40px;
	color: rgb(0, 187, 255);
}
QLineEdit {
	background: transparent;
	border: 0px solid transparent;
}

QSlider:horizontal {
	background: none;
}
QSlider::groove:horizontal {
	border: 0px solid transparent;
	background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 255, 160), stop:0.55 rgba(255, 255, 255, 60), stop:1 rgba(255, 255, 255, 255));
	height: 2px;
}
QSlider::handle:horizontal {
	width: 15px;
	height: 9px;
	border-radius: 4px;
	margin-top: -5px;
	margin-bottom: -5px;
	border: 2px solid rgb(0, 133, 181);
	background-color: rgb(0, 187, 255);
}

QDoubleSpinBox {
	padding: 2px;
	padding-top: 0px;
	color: white;
	font: 10px;
}
QDoubleSpinBox, QSpinBox, QLineEdit  {
	padding: 3px;
	color: white;
}
QSpinBox::up-button, QDoubleSpinBox::up-button {
	width: 5px;
    image: url(:/icons/res_icon_plus.png);
	padding: 2px;
	border: 0px solid transparent;
	background: rgba(255, 255, 255, 64);
}
QSpinBox::down-button, QDoubleSpinBox::down-button {
	width: 5px;
    image: url(:/icons/res_icon_minus.png);
	padding: 2px;
	border: 0px solid transparent;
	background: rgba(255, 255, 255, 64);
}

QLabel#lastMessage {
	font: 11px &quot;Lucida Console&quot;, &quot;Monaco&quot;;
	color: rgb(167, 167, 167);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="globalFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="transportFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-radius: 20px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="ffButton">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Fast rewind</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="IanniX.qrc">
             <normaloff>:/general/res_icon_ff.png</normaloff>:/general/res_icon_ff.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="playButton">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Play/pause</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="IanniX.qrc">
             <normaloff>:/general/res_icon_play.png</normaloff>
             <normalon>:/general/res_icon_pause.png</normalon>:/general/res_icon_play.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="timeFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QLineEdit" name="timeEdit">
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Elapsed time</string>
           </property>
           <property name="inputMask">
            <string notr="true">999:99.999; </string>
           </property>
           <property name="text">
            <string notr="true">444:44.444</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="speedFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lastMessage">
           <property name="text">
            <string>No message</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QSlider" name="speedSlider">
             <property name="minimum">
              <number>-500</number>
             </property>
             <property name="maximum">
              <number>500</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="speedSpin">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Playback speed</string>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-999.000000000000000</double>
             </property>
             <property name="maximum">
              <double>999.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="perfFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9px;
color: rgba(255,255,255,128);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="perfSchedulerEdit">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Scheduler accuracy</string>
           </property>
           <property name="inputMask">
            <string notr="true">009 ms; </string>
           </property>
           <property name="text">
            <string notr="true"> ms</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="perfOpenGLEdit">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Graphism accuracy</string>
           </property>
           <property name="inputMask">
            <string notr="true">009 fps; </string>
           </property>
           <property name="text">
            <string notr="true"> fps</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="perfCpuEdit">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>CPU usage</string>
           </property>
           <property name="inputMask">
            <string notr="true">009 %; </string>
           </property>
           <property name="text">
            <string notr="true"> %</string>
           </property>
           <property name="cursorPosition">
            <number>5</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="logoFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border: 0px solid transparent;
	background-color: transparent;
}
QPushButton:hover {
	border: 1px solid rgb(0, 187, 255);
	background-color: rgb(0, 137, 241, 128);
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="logoButton">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>50</height>
            </size>
           </property>
           <property name="toolTip">
            <string>About IanniX</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="IanniX.qrc">
             <normaloff>:/general/res_logo.png</normaloff>:/general/res_logo.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>45</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ffButton</tabstop>
  <tabstop>playButton</tabstop>
  <tabstop>timeEdit</tabstop>
  <tabstop>speedSlider</tabstop>
  <tabstop>speedSpin</tabstop>
  <tabstop>perfSchedulerEdit</tabstop>
  <tabstop>perfOpenGLEdit</tabstop>
  <tabstop>perfCpuEdit</tabstop>
  <tabstop>logoButton</tabstop>
 </tabstops>
 <resources>
  <include location="IanniX.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>playButton</sender>
   <signal>released()</signal>
   <receiver>UiTransport</receiver>
   <slot>actionPlay_pause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>-59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ffButton</sender>
   <signal>released()</signal>
   <receiver>UiTransport</receiver>
   <slot>actionFast_rewind()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>-138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logoButton</sender>
   <signal>released()</signal>
   <receiver>UiTransport</receiver>
   <slot>actionLogo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>761</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>663</x>
     <y>-88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>UiTransport</receiver>
   <slot>actionGoto()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>-68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>perfSchedulerEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>UiTransport</receiver>
   <slot>actionSetScheduler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>707</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>perfOpenGLEdit</sender>
   <signal>selectionChanged()</signal>
   <receiver>UiTransport</receiver>
   <slot>actionSetOpenGL()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>707</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>641</x>
     <y>-105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>perfSchedulerEdit</sender>
   <signal>lostFocus()</signal>
   <receiver>UiTransport</receiver>
   <slot>actionSetScheduler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>707</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>-127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>perfOpenGLEdit</sender>
   <signal>lostFocus()</signal>
   <receiver>UiTransport</receiver>
   <slot>actionSetOpenGL()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>687</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>678</x>
     <y>-109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speedSpin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>UiTransport</receiver>
   <slot>actionSpeed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>630</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>563</x>
     <y>-74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speedSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>UiTransport</receiver>
   <slot>actionSpeedSlider()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>-125</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>actionPlay_pause()</slot>
  <slot>actionFast_rewind()</slot>
  <slot>actionLogo()</slot>
  <slot>actionGoto()</slot>
  <slot>actionSetScheduler()</slot>
  <slot>actionSetOpenGL()</slot>
  <slot>actionSpeedSlider()</slot>
  <slot>actionSpeed()</slot>
  <slot>actionPresentation()</slot>
 </slots>
</ui>
